src/Physics/HadronTransport/2025/HAIntranuke2025.cxx

```diff
@@ -0,0 +1,1631 @@
+//____________________________________________________________________________
+/*
+ Copyright (c) 2003-2024, The GENIE Collaboration
+ For the full text of the license visit http://copyright.genie-mc.org
+ 
+
+ Author: Steve Dytman <dytman+@pitt.edu>, Pittsburgh Univ.
+         Aaron Meyer <asm58@pitt.edu>, Pittsburgh Univ.
+         Alex Bell, Pittsburgh Univ.
+         Hugh Gallagher <gallag@minos.phy.tufts.edu>, Tufts Univ.
+         Costas Andreopoulos <c.andreopoulos \at cern.ch>, Rutherford Lab.
+         September 20, 2005
+
+ For the class documentation see the corresponding header file.
+
+ Important revisions after version 2.0.0 :
```

Comment:
Are these revision comments accurate for this file?

I suspect that the boilerplate comments for all the files in this PR need to be reviewed for relevancy and completeness.